nebula_ros/include/nebula_ros/continental/multi_continental_ars548_hw_interface_ros_wrapper.hpp

```diff
@@ -0,0 +1,146 @@
+// Copyright 2024 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NEBULA_ContinentalARS548HwInterfaceRosWrapper_H
+#define NEBULA_ContinentalARS548HwInterfaceRosWrapper_H
+
+#include <ament_index_cpp/get_package_prefix.hpp>
+#include <boost_tcp_driver/tcp_driver.hpp>
+#include <nebula_common/nebula_common.hpp>
+#include <nebula_hw_interfaces/nebula_hw_interfaces_continental/multi_continental_ars548_hw_interface.hpp>
+#include <nebula_ros/common/nebula_hw_interface_ros_wrapper_base.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_components/register_node_macro.hpp>
+
+#include <geometry_msgs/msg/accel_with_covariance_stamped.hpp>
+#include <geometry_msgs/msg/twist_with_covariance_stamped.hpp>
+#include <nebula_msgs/msg/nebula_packet.hpp>
+#include <nebula_msgs/msg/nebula_packets.hpp>
+#include <std_msgs/msg/bool.hpp>
+#include <std_msgs/msg/float32.hpp>
+
+#include <boost/asio.hpp>
+
+#include <memory>
+#include <mutex>
+#include <string>
+#include <thread>
+#include <unordered_map>
+#include <vector>
+
+namespace nebula
+{
+namespace ros
+{
+/// @brief Get parameter from rclcpp::Parameter
+/// @tparam T
+/// @param p Parameter from rclcpp parameter callback
+/// @param name Target parameter name
+/// @param value Corresponding value
+/// @return Whether the target name existed
+template <typename T>
+bool get_param(const std::vector<rclcpp::Parameter> & p, const std::string & name, T & value)
+{
+  auto it = std::find_if(p.cbegin(), p.cend(), [&name](const rclcpp::Parameter & parameter) {
+    return parameter.get_name() == name;
+  });
+  if (it != p.cend()) {
+    value = it->template get_value<T>();
+    return true;
+  }
+  return false;
+}
+
+/// @brief Hardware interface ros wrapper of continental radar ethernet driver
+/// NOTE: this is a temporal class that acts as a single hw interface for multiple devices
```

Comment:
(pedantic) temporary

Reply:
Addressed in [764e584](https://github.com/tier4/nebula/pull/123/commits/764e584d1e9caa722e73307123f48223a51e265a)